README.md

```diff
@@ -56,6 +56,20 @@ public class PingCommand extends SlashCommand {
 
 For more information on how this works, visit the [DIH4JDA Wiki!](https://github.com/DynxstyGIT/DIH4JDA/wiki)
 
+# API Documentation
+
+### `GET guilds/{guild_id}/metrics` 
+- Responds with guild-specific metrics, such as the member- and (approximate) online count.
+
+### `GET guilds/{guild_id}/users/{user_id}`
+- Responds with the specified users' profile which includes some basic info, such as the users' name and avatar url, but also their recent warns, their current help experience and more!
+
+### `GET guilds/{guild_id}/leaderboard/qotw?page=1`
+- A paginated endpoint which responds with an ordered list of users, based on their QOTW points.
+
+### `GET guilds/{guild_id}/leaderboard/experience?page=1`
+- A paginated endpoint which responds with an ordered list of users, based on their help channel experience.
+
```

Comment:
Wec could also mention that there is a publicly available hosted version available at `api.javadiscord.net`. Maybe also just mention our guild id in that context but that's not really necessary